src/components/common/ErrorMessage.css

```diff
@@ -25,22 +23,42 @@ limitations under the License.
 .ErrorMessage--msg {
   color: #c00;
   margin: 0;
-  padding: 1rem;
+  padding-bottom: 1rem;
 }
 
 .ErrorMessage--details {
+  background: #f5f5f5;
+  border: 1px solid #e6e6e6;
```

Comment:
Is there a way to reuse things like border? Seems like its not specific to the message details, but just a general UI theme we use. Suggestions: Use a utility class (basscss), css vars, or styled components in JS

Reply:
CSS vars aren't supported in IE11, which is ~3% of browsers. I try to avoid stuff that's not supported by IE. 

Another approach might be to embrace a CSS preprocessor now that we have react-app-rewired going. What do you think of PostCSS with [these plugins](https://jf.io/2015/10/26/a-selection-of-postcss-plugins#plugin-selection)?

Reply:
I'm good with that. We need something to manage this CSS better.

Reply:
Created a ticket for this #200